modules/gha/dagger.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "gha",
+  "sdk": "go",
+  "dependencies": [
+    {
+      "name": "wolfi",
+      "source": "github.com/shykes/daggerverse/wolfi@daae822bfa6d67e9e241b92ebe43e029e0c2c043"
```

Comment:
Should we depend on the local Wolfi module?

It will make iterating on changes quicker, same as for the `gha` module.

Reply:
Done :tada: